### PR TITLE
Added the Ascent as a content module.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
- Readds the Ascent (alates, gynes and serpentids) as a content module, including gear and submap.
- Compared to Baycode, they don't have their special grabs and monarch serpentid queens are totally removed.
- Since there are no nabbers, renames the MSW as just 'serpentids'.
- Removes some references to the Skrell and Unathi.

### CODE REVIEW

File: code/content_packages/ascent/datum/codex.dm [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/datum/codex_entry/ascent
+	display_name = "The Ascent"
+	associated_strings = list("ascent")
+	associated_paths = list(
+		/mob/living/silicon/robot/flying/ascent,
+		/obj/item/multitool/mantid,
+		/obj/item/weldingtool/electric/mantid,
+		/obj/item/cell/mantid,
+		/obj/item/gun/energy/particle,
```
Comment (comma) [inline code/content_packages/ascent/datum/codex.dm:9]: can you move these guns here too while you're at it?

Comment (MistakeNot4892) [reply code/content_packages/ascent/datum/codex.dm:9]: I thought I already did.

Comment (MistakeNot4892) [reply code/content_packages/ascent/datum/codex.dm:9]: https://github.com/NebulaSS13/Nebula/pull/161/files#diff-83e7ab0cf4664cd8fe84fa6099e23c58R1

**Comment by MistakeNot4892** [comment]
Ready to go.

**Review by comma — APPROVED** [review]

**Pull request merged**